### ISSUE TITLE
#sugar-newbies is not a channel we use

### ISSUE THREAD

**Issue by quozl** [sugarlabs/www-sugarlabs#188]
@yashagrawal3 told me that `#sugar-newbies` is [offered](https://sugarlabs.org/about-irc/), but this is not a channel in our [list of channels](https://wiki.sugarlabs.org/go/Internet_Relay_Chat).

It would be no surprise if there is no response to questions from experts, if others don't know about it, so I'd like this channel deprecated in favour of `#sugar`.

Discussion may ensue ...

**Comment by walterbender** [comment]
I am all for sticking to #sugar We can manage the traffic (even during GCI), and I think it is useful for everyone to see newbie questions.

**Comment by vipulgupta2048** [comment]
Finding this issue, actually makes me giggle. I have been hanging out on that channel from the start, and often wonder why no is there. One fine day, I found that someone did reply. Here's a [screenshot](https://imgur.com/a/kNLw9). I joined on March 18, 2018. There has been a total of 3 messages from then. 

`#sugar` is good enough place for discussion by both beginners and experts, even newcomers. I am quite active there for discussions and other people are too. Hence, the channel #sugar-newbies can be safely depreciated. Also, we should actually **find** where #sugar-newbies has been mentioned in the first place. Because I would have found it too somewhere. Searching ...
